### ISSUE TITLE
`samples` keyword arg ignored in `FreqDist.tabulate`

### ISSUE THREAD

**Issue by rdbliss** [nltk/nltk#2356]
It looks like the `samples` keyword argument mentioned in the docstring of `FreqDist.tabulate()` is ignored. Is this the intended behavior?

Here are the arguments from [the docstring](https://github.com/nltk/nltk/blob/1805fe870635afb7ef16d4ff5373e1c3d97c9107/nltk/probability.py#L308):

>:param samples: The samples to plot (default is all samples)
>:type samples: list
>:param cumulative: A flag to specify whether the freqs are cumulative (default = False)
>:type title: bool

Here's an example of `samples` being ignored:

```python
from nltk import FreqDist

fd = FreqDist(["foo", "bar"])
# I expect to get just "foo", but get "foo" *and* "bar".
fd.tabulate(samples=["foo"])
```

This is different from what `ConditionalFreqDist.tabulate()` does, so I'm not sure what's intended here. Either way, it would be nice for the docstring to be clearer.

**Comment by anantvir** [comment]
Can I start working on this ?

**Comment by stevenbird** [comment]
Thanks @anantvir.
(Resolved in #2469)
